zeq/publisher.h

```diff
@@ -17,17 +19,70 @@ namespace detail { class Publisher; }
 /**
  * Serves and publishes events, consumed by Subscriber.
  *
+ * The session is tied to ZeroConf announcement and can be disabled with
+ * zeq::NO_ANNOUNCE_SESSION.
```

Comment:
"passing zeq::NO_ANNOUNCE_SESSION as the session name." At least that's what I guessed.